src/library/scala/collection/immutable/LazyList.scala

```diff
@@ -43,29 +58,21 @@ import scala.runtime.Statics
   *  // 2
   *  // 3
   *  }}}
-  * 
-  *  A `LazyList`, like the one in the example above, may be infinite in length.
-  *  Aggregate methods, such as `count`, `sum`, `max` or `min` on such infinite length 
-  *  sequences will not terminate. Filtered infinite lazy lists are also effectively
-  *  infinite in length.
   *
-  *  Elements of a `LazyList` are memoized; that is, the value of each element
-  *  is computed only once.
-  *  To illustrate, we will alter body of the `fibs` value above and take some
-  *  more values:
+  *  To illustrate, let's add some output to the definition `fibs`, so we
+  *  see what's going on.
   *
   *  {{{
   *  import scala.math.BigInt
   *  object Main extends App {
-  *
-  *    val fibs: LazyList[BigInt] = BigInt(0) #:: BigInt(1) #:: fibs.zip(
-  *      fibs.tail).map(n => {
-  *        println("Adding %d and %d".format(n._1, n._2))
-  *        n._1 + n._2
-  *      })
-  *
-  *    fibs take 5 foreach println
-  *    fibs take 6 foreach println
+  *    val fibs: LazyList[BigInt] =
+  *      BigInt(0) #:: BigInt(1) #::
+  *        fibs.zip(fibs.tail).map{n =>
```

Comment:
I feel like it's usually considered preferred to have spaces around braces (except in interpolations)?

```suggestion
  *        fibs.zip(fibs.tail).map { n =>
```

Reply:
I'm going with `.map{ n => ...` (space after brace, but not before), but I'm not really sure what the range of "acceptable" styles is and I wouldn't mind if someone changed it again in a future PR.